tools/src/publish/utils.ts

```diff
@@ -0,0 +1,101 @@
+import { execFileSync, execSync } from 'node:child_process';
+import { join } from 'node:path';
+import { objectToCliArgs } from '../../../packages/utils/src';
+import { BUMP_SCRIPT } from './constants';
+import { PublishOptions } from './types';
+
+export function nxRunManyPublish({
+  registry,
+  tag = 'e2e',
+  nextVersion,
+}: PublishOptions) {
+  console.info(`Publish packages to registry: ${registry}.`);
+
+  execFileSync(
+    'npx',
+    [
+      'nx',
+      'run-many',
+      '--targets=publish',
+      '--parallel=1',
+      '--',
+      ...(nextVersion ? [`--nextVersion=${nextVersion}`] : []),
+      ...(tag ? [`--tag=${tag}`] : []),
+      ...(registry ? [`--registry=${registry}`] : []),
+    ],
+    { env: process.env, stdio: 'inherit', shell: true },
+  );
+}
```

Comment:
Previously located in [tools/scripts/publish.plugin.ts](https://github.com/code-pushup/cli/pull/798/files#r1722239486)